drivers/wifi/eswifi/eswifi_socket.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2019 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <kernel.h>
+#include <device.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+#include "eswifi.h"
+#include <net/net_pkt.h>
+
+#define LOG_LEVEL CONFIG_WIFI_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(wifi_eswifi_common);
```

Comment:
You can also write this into one line like this so the LOG_LEVEL do not need to be defined.
```
LOG_MODULE_REGISTER(wifi_eswifi, CONFIG_WIFI_LOG_LEVEL);
```
Also the module can be set to `wifi_eswifi`

Please put the logging include before the zephyr.h. That way if some of the header files have inline functions with logging, the output is printed properly.